pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -105,7 +105,9 @@ var _ = Describe("Pod Network", func() {
 	}
 
 	BeforeEach(func() {
-		cacheFactory = fake.NewFakeInMemoryNetworkCacheFactory()
+		tmpDir, err := ioutil.TempDir("/tmp", "interface-cache")
```

Comment:
I think that you override the `tmpDir` here. The `AfterEach` may not be able to clean up the `tmpDir`.

Reply:
Done